src/components/scrollToTop/ScrollToTop.scss

```diff
@@ -0,0 +1,19 @@
+.scroll-to-top{
+  position: fixed; 
+  width: 56px;
+  height: 56px;
+  bottom: 4%;
+  right: 2%;
+  z-index: 1000;
+}
+
+@media only screen and (max-width: 1000px) {
+  .scroll-to-top{
+      position: fixed; 
+      width: 56px;
+      height: 56px;
+      bottom: 22%;
+      right: 5%;
```

Comment:
media query is great! I played with the numbers and I think these make the position a little lower and centered above the other button. Can you try this?

bottom: 11%;
right: 7.5%

Reply:
sure, but I can't do it today. I will fix them tmr.

Reply:
No rush :)